exchanges/deribit/deribit.go

```diff
@@ -0,0 +1,2716 @@
+package deribit
```

Comment:
There are a couple of functions added since you initially did this:
https://docs.deribit.com/#public-exchange_token
https://docs.deribit.com/#public-fork_token
https://docs.deribit.com/#private-logout
https://docs.deribit.com/#private-enable_cancel_on_disconnect
https://docs.deribit.com/#private-disable_cancel_on_disconnect
https://docs.deribit.com/#private-get_cancel_on_disconnect
https://docs.deribit.com/#public-hello
https://docs.deribit.com/#public-status
https://docs.deribit.com/#public-unsubscribe_all
https://docs.deribit.com/#private-unsubscribe_all